sphinx/ext/autodoc/__init__.py

```diff
@@ -297,6 +292,14 @@ def __init__(self, directive, name, indent=u''):
         # the module analyzer to get at attribute docs, or None
         self.analyzer = None        # type: Any
 
+    @property
+    def documenters(self):
+        # type: () -> Dict[unicode, Type[Documenter]]
+        """Returns registered Documenter classes"""
+        classes = dict(AutoDirective._registry)  # registered directly
```

Comment:
oh, I noticed this is same as `get_documenters()`. So I'll replace this with it.